pkg/vm/engine/tae/logtail/operator.go

```diff
@@ -59,14 +60,22 @@ func (op *BoundOperator) Run() (err error) {
 		}
 		for _, dirtySeg := range tblDirty.Segs {
 			if seg, err = tbl.GetSegmentByID(dirtySeg.ID); err != nil {
-				return err
+				if moerr.IsMoErrCode(err, moerr.ErrNotFound) {
```

Comment:
What is the frequency of this not found?   If it is really often, please considered an Ok...EOB/END code.   Basically we if not found is the common case, we want an Ok code.

Reply:
the frequency is related to the flush interval option. GetSegmentByID etc. are not just only used in collecting logtail, so here it is better to ignore the not found err and do not change the code returned by GetSegmentByID etc

Reply:
ErrNotFound is replaced with GetOK.... 
It should be modified when #6166 is merged into the main.